physics/satmedmfvdifq.F

```diff
@@ -929,6 +971,31 @@ subroutine satmedmfvdifq_run(im,km,progsigma,ntrac,ntcw,ntrw,     &
      &    thlx,thvx,thlvx,gdx,thetae,
      &    krad,mrad,radmin,buod,xmfd,
      &    tcdo,qcdo,ucdo,vcdo,xlamde,bl_dnfr)
+
+      if (tc_pbl == 1) then
```

Comment:
Could you indent the code inside the `if (tc_pbl == 1) then` statement for readability?

Reply:
Changed accordingly. Thanks, @grantfirl!